From 81092fd7f446ac4ae7d81433552c4a7da7598e16 Mon Sep 17 00:00:00 2001
From: Chen Chen <cncn@google.com>
Date: Tue, 5 Oct 2021 17:46:02 -0700
Subject: [PATCH] BluetoothSecurity: Add BLUETOOTH_PRIVILEGED permission for
 pairing dialog

Bug: 194300867
Test: Build

Change-Id: I5a496df50550ea7ee6986c960e28ae3e4a056b4b
(cherry picked from commit e5ccfba9962f22d05cbf9cc6097ef7b74938ecf0)
---
 AndroidManifest.xml | 1 +
 1 file changed, 1 insertion(+)

diff --git a/AndroidManifest.xml b/AndroidManifest.xml
index 2f53cc1e7c..a9f0a3dd7a 100644
--- a/AndroidManifest.xml
+++ b/AndroidManifest.xml
@@ -2014,6 +2014,7 @@
         </activity>
 
         <activity android:name=".bluetooth.BluetoothPairingDialog"
+                  android:permission="android.permission.BLUETOOTH_PRIVILEGED"
                   android:excludeFromRecents="true"
                   android:windowSoftInputMode="stateVisible|adjustResize"
                   android:theme="@style/Theme.AlertDialog"
-- 
2.34.0.rc2.393.gf8c9666880-goog

